.github/workflows/dependabot-auto-merge.yml

```diff
@@ -5,10 +5,11 @@ on:
 
 jobs:
   auto-merge:
+    if: github.repository_owner == 'andrewthetechie'
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: ahmadnassri/action-dependabot-auto-merge@v2
         with:
           target: minor
-          github-token: ${{ secrets.THIS_PAT }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think this one will have to stay using the PAT, because it comments on the PR as "me" so an approved writer merges the commit.

Reply:
Understood, if you keep the if repository_owner statement it solves the problem without changing the token